duckari/duck.py

```diff
@@ -25,13 +26,25 @@ def decorate(func: _ListenerT) -> _ListenerT:
 
         return decorate
 
+    def command(self, name: str = None): # TODO: Add return type
+        def decorate(func: _ListenerT) -> _ListenerT:
```

Comment:
A command is not a listener. This type hint is wrong.

Reply:
I'm not so good at type hints. I'll just remove it as I have no idea what to put.